.github/ISSUE_TEMPLATE/api_bug.md

```diff
@@ -0,0 +1,117 @@
+---
+name: API Bug
+about: "Report a bug you have encountered using SeAT API."
+---
+
+# Bug
+*This template contains an example bug report. Please replace all text except for the checklist and the section headers (they start with \#). Thank you for your contribution to improving SeAT.*
+
+*In this space, please provide a short description of the bug you've encountered.
+This should help us to understood what you've did while you get the error. E.g.:*
+
+When I'm sending a request to `/roles/affiliation/character` I got a success response but the user still not have the role attached to him.
+
+*Side note: If you can, please attach a gist version or a link to a repository where is your script.*
```

Comment:
`where is your script`: I'd suggest following wording:

'Side note: If you are using SeAT API programmatically, please attach a gist version or a link to your repository.'

(Note: In a formal letter one usually refers from using abbreviations. Which i have seen you using. However, i did not correct them)